deploy-retool

```diff
@@ -218,22 +218,26 @@ echo " -- ${GREEN}${BOLD}!! RETOOL IS BOOTING !!${NORMAL} --"
 
 WAITED=0
 
-api_running() {
-  API_DOCKER_CONTAINER="$($MAYBE_SUDO docker ps | grep 'retool-onpremise_api')"
+api_container_up() {
+  API_DOCKER_CONTAINER = "$($MAYBE_SUDO docker ps | grep 'retool-onpremise-api')"
```

Comment:
why did we change from underscore to hyphen? somewhat concerned about that.

Reply:
because the name shows up as hyphen when I run docker ps. not sure why it was underscore in the first place actually

Reply:
shows up as an underscore for me... maybe we should make it a wild card character to be safe? 


<img width="342" alt="image" src="https://user-images.githubusercontent.com/42283515/200717597-f26ad0db-d3fb-4792-b879-cbafccb22741.png">

Reply:
i think different docker versions will format it differently 🤔